plugin.go

```diff
@@ -162,6 +176,15 @@ func color(build Build) string {
 	}
 }
 
+func checkAndSelectColor(color string) string {
```

Comment:
this function should be removed.  Slack supports color labels and also supports hex values. If the user inputs an invalid value, the slack API will return an error. No need to validate this client-side.